client/containers/DashboardOverview/overviewRows/PubOverviewRow.tsx

```diff
@@ -38,16 +76,40 @@ const PubOverviewRow = (props: Props) => {
 			target="_blank"
 		/>
 	);
+
+	// for when a pub has a submission but is being rendered in collection or community overview
+	if (pub.submission) {
+		const { status } = pub.submission;
+		const iconLabelPairs = isSubmission
+			? labelPairs([getSubmissionStatusLabel(status), getSubmissionTimeLabel(pub)])
+			: labelPairs([
+					...getScopeSummaryLabels(pub.scopeSummary),
+					getPubReleasedStateLabel(pub),
+			  ]);
+		return (
```

Comment:
Removing duplication here will make it easier for readers to understand how these two codepaths diverge, and to make future changes. As far as I can tell the only difference between this `<OverviewRowSkeleton>` and the one rendered underneath is the `iconLabelPairs` prop. Let's hone in on that — we need a function like:

```ts
renderIconLabelPairs(pub: Pub, hasSubmission: boolean): React.ReactNode
```

The conditional logic you've added here should be contained in this function, which will return the thing that we can unconditionally pass into `<OverviewRowSkeleton>`.